### ISSUE TITLE
False positive IDE0060 on VB xml syntax

### ISSUE THREAD

**Issue by huoyaoyuan** [dotnet/roslyn#39095]
**Version Used**: 
Visual Studio 16.3.2

**Steps to Reproduce**:
Found at https://github.com/dotnet/roslyn/blob/7f60b3849b6dd1c3ee64cd2d327cfc5ea6fa7b64/src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb#L235-L247

Minimal repro:
```vb
Module Program
    Function M(xml As XElement) As Object
        Return xml.@name
    End Function
End Module
```

**Expected Behavior**:
The xml element is treated as used.
**Actual Behavior**:
IDE0060 reported.

**Comment by huoyaoyuan** [comment]
Duplicate of #37988. But that fix should already be merged in 16.3 right?

**Comment by mavasani** [comment]
@huoyaoyuan I can't repro this on our latest internal dogfood build. Can you try on latest 16.4 preview build?

**Comment by huoyaoyuan** [comment]
@mavasani I haven't install preview. Trusting your internal test and unit test in master.
After comparing, the fix is in 16.3 milestone, but not merged into release/dev16.3. A bit confused about this.

**Comment by jinujoseph** [comment]
Duplicate of #37988
